test/coverage-test/vitest.workspace.custom.ts

```diff
@@ -78,6 +78,29 @@ export default defineWorkspace([
       ],
     },
   },
+  {
+    test: {
+      ...config.test,
+      name: 'v8-browser',
+      env: { COVERAGE_PROVIDER: 'v8', COVERAGE_BROWSER: 'true' },
+      include: [
+        BROWSER_TESTS,
+
+        // Other non-provider-specific tests that should be run on browser mode as well
+        '**/ignore-hints.test.ts',
+        '**/import-attributes.test.ts',
+        '**/multi-suite.test.ts',
+        '**/setup-files.test.ts',
+        '**/results-snapshot.test.ts',
+        '**/reporters.test.ts',
+        '**/temporary-files.test.ts',
+        '**/test-reporter-conflicts.test.ts',
+
+        // TODO: Enable, last source map issue here
+        // '**/vue.test.ts',
```

Comment:
Need to figure out what's going on here. The `Defined.vue` is the only file that is transformed differently on Node vs Browser.

https://github.com/vitest-dev/vitest/blob/7b37d27ba7ce9e75fce385126e61d85d575f82ad/test/coverage-test/fixtures/src/Vue/Defined.vue#L1-L22

On browser mode it returns:

```js
import { defineComponent as _defineComponent } from "/node_modules/.vite/deps/vue.js?v=256f6d3b";
const defined = "hello";
const _sfc_main = /* @__PURE__ */ _defineComponent({
  __name: "Defined",
  setup(__props, { expose: __expose }) {
    __expose();
    if (defined) {
      console.log("Covered condition");
    } else {
      console.log("Uncovered condition");
    }
    const __returned__ = { defined };
    Object.defineProperty(__returned__, "__isScriptSetup", { enumerable: false, value: true });
    return __returned__;
  }
});
import { toDisplayString as _toDisplayString } from "/node_modules/.vite/deps/vue.js?v=256f6d3b";
function _sfc_render(_ctx, _cache, $props, $setup, $data, $options) {
  return _toDisplayString($setup.defined);
}
import "/fixtures/src/Vue/Defined.vue?vue&type=style&index=0&scoped=f7f04e08&lang.css";
import _export_sfc from "/@id/__x00__plugin-vue:export-helper";
export default /* @__PURE__ */ _export_sfc(_sfc_main, [["render", _sfc_render], ["__scopeId", "data-v-f7f04e08"], ["__file", "/Users/x/vitest/test/coverage-test/fixtures/src/Vue/Defined.vue"]]);
```

On Node with `happy-dom` it returns:
```js
const __vite_ssr_import_0__ = await __vite_ssr_import__("/@fs/Users/x/vitest/node_modules/.pnpm/vue@3.4.35_typescript@5.5.4/node_modules/vue/index.mjs", {"importedNames":["defineComponent"]});
const __vite_ssr_import_1__ = await __vite_ssr_import__("/@fs/Users/x/vitest/node_modules/.pnpm/vue@3.4.35_typescript@5.5.4/node_modules/vue/index.mjs", {"importedNames":["toDisplayString"]});
const __vite_ssr_import_2__ = await __vite_ssr_import__("/fixtures/src/Vue/Defined.vue?vue&type=style&index=0&scoped=f7f04e08&lang.css");
const __vite_ssr_import_3__ = await __vite_ssr_import__("/@id/__x00__plugin-vue:export-helper", {"importedNames":["default"]});

const defined = "hello";
const _sfc_main = /* @__PURE__ */ __vite_ssr_import_0__.defineComponent({
  __name: "Defined",
  setup(__props, { expose: __expose }) {
    __expose();
    if (defined) {
      console.log("Covered condition");
    } else {
      console.log("Uncovered condition");
    }
    const __returned__ = { defined };
    Object.defineProperty(__returned__, "__isScriptSetup", { enumerable: false, value: true });
    return __returned__;
  }
});

function _sfc_render(_ctx, _cache, $props, $setup, $data, $options) {
  return __vite_ssr_import_1__.toDisplayString($setup.defined);
}


__vite_ssr_exports__.default = /* @__PURE__ */ __vite_ssr_import_3__.default(_sfc_main, [["render", _sfc_render], ["__scopeId", "data-v-f7f04e08"], ["__file", "/Users/x/vitest/test/coverage-test/fixtures/src/Vue/Defined.vue"]]);

//# sourceMappingSource=vite-node
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJtYXBwaW5ncyI6IkFBQUEsQ0FBQTs7Ozs7QUFDQSxNQUFNLFVBQVU7Ozs7O0FBR2hCLFFBQUksU0FBUztBQUNYLGNBQVEsSUFBSSxtQkFBbUI7QUFBQSxJQUNqQyxPQUNLO0FBQ0gsY0FBUSxJQUFJLHFCQUFxQjtBQUFBLElBQ25DOzs7Ozs7OzsrQ0FJSyxjQUFPIiwibmFtZXMiOltdLCJpZ25vcmVMaXN0IjpbXSwic291cmNlcyI6WyJEZWZpbmVkLnZ1ZSJdLCJzb3VyY2VzQ29udGVudCI6WyI8c2NyaXB0IHNldHVwIGxhbmc9XCJ0c1wiPlxuY29uc3QgZGVmaW5lZCA9ICdoZWxsbydcblxuLyogZXNsaW50LWRpc2FibGUgbm8tY29uc29sZSAqL1xuaWYgKGRlZmluZWQpIHtcbiAgY29uc29sZS5sb2coJ0NvdmVyZWQgY29uZGl0aW9uJylcbn1cbmVsc2Uge1xuICBjb25zb2xlLmxvZygnVW5jb3ZlcmVkIGNvbmRpdGlvbicpXG59XG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICB7eyBkZWZpbmVkIH19XG48L3RlbXBsYXRlPlxuXG48IS0tIFN0eWxlIGJsb2NrIHRyaWdnZXJzIGEgc3BlY2lmaWMgY29uZGl0aW9uIHdoZXJlIHNvdXJjZW1hcHMgdXNlZCB0byBicmVhayByYW5kb21seSAtLT5cbjxzdHlsZSBsYW5nPVwiY3NzXCIgc2NvcGVkPlxuYm9keSB7XG4gIGJhY2tncm91bmQtY29sb3I6ICNGRkY7XG59XG48L3N0eWxlPlxuIl0sImZpbGUiOiIvZml4dHVyZXMvc3JjL1Z1ZS9EZWZpbmVkLnZ1ZSJ9',
```

That looks like SSR transformed even though `happy-dom` should set `transformMode: 'web'` 🤔